cfme/fixtures/pytest_selenium.py

```diff
@@ -325,7 +325,7 @@ def _nothing_in_flight():
         silent_failure=True)
 
     # If we are not supposed to take page screenshots...well...then...dont.
-    if not store.config.getvalue('page_screenshots'):
+    if store.has_config and not store.config.getvalue('page_screenshots'):
```

Comment:
So, based on the previous comment:
`if store.config and not store.config.getvalue('page_screenshots'):`